dotnet/Gherkin.NuGetPackages/Gherkin.nuspec

```diff
@@ -1,19 +1,22 @@
-<?xml version="1.0"?>
-<package xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
-  <metadata xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
-    <id>Gherkin</id>
-    <version>3.0.0-alpha2</version>
-    <title>Gherkin Parser</title>
-    <authors>Aslak Hellesøy, Gaspar Nagy, TechTalk</authors>
-    <owners>TechTalk</owners>
-    <summary>Cross-platform parser for the Gherkin language.</summary>
-    <description>Cross-platform parser for the Gherkin language, used by Cucumber and SpecFlow to parse .feature files.</description>
-    <language>en-US</language>
-    <projectUrl>https://github.com/cucumber/gherkin3</projectUrl>
-    <iconUrl>http://go.specflow.org/gherkin-nuget-icon</iconUrl>
-    <licenseUrl>https://github.com/cucumber/gherkin3/blob/master/LICENSE</licenseUrl>
-    <requireLicenseAcceptance>false</requireLicenseAcceptance>
-    <copyright>Copyright © 2014-2015 Aslak Hellesøy, Gaspar Nagy, TechTalk</copyright>
-    <tags>specflow gherkin cucumber</tags>
-  </metadata>
+<?xml version="1.0"?>
+<package xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <metadata xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+    <id>Gherkin</id>
+    <version>3.0.0-alpha2</version>
+    <title>Gherkin Parser</title>
+    <authors>Aslak Hellesøy, Gaspar Nagy, TechTalk</authors>
+    <owners>TechTalk</owners>
+    <summary>Cross-platform parser for the Gherkin language.</summary>
+    <description>Cross-platform parser for the Gherkin language, used by Cucumber and SpecFlow to parse .feature files.</description>
+    <language>en-US</language>
+    <projectUrl>https://github.com/cucumber/gherkin3</projectUrl>
+    <iconUrl>http://go.specflow.org/gherkin-nuget-icon</iconUrl>
```

Comment:
Our [brand](https://github.com/cucumber-ltd/brand) has changed, and that logo is obsolete. Please use https://cucumber.io/images/cucumber-mark-32x32.png